waspc/data/Generator/templates/react-app/README.md

```diff
@@ -1,68 +1,21 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This project was bootstrapped with [Vite's React TS template](https://github.com/vitejs/vite/tree/main/packages/create-vite/template-react-ts).
 
 ## Available Scripts
 
 In the project directory, you can run:
 
 ### `npm start`
 
-Runs the app in the development mode.<br>
+Runs the app in development mode.<br>
 Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
 
 The page will reload if you make edits.<br>
 You will also see any lint errors in the console.
 
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br>
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
 ### `npm run build`
 
 Builds the app for production to the `build` folder.<br>
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
 The build is minified and the filenames include the hashes.<br>
 Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
```

Comment:
VIte does not have an equivalent of these things. Is that ok for us @Martinsos? Ejection was once a semi-big deal.

Reply:
Aha interesting!
Well people can still take the whole generated code with themselves and go, so I think ejection in that sense is still there, it is just that ejecting from the Vite itself is not there which is ok. So I think this does not affect ejection from Wasp.
I would though say that even ejection from Wasp is not that crucial anymore, but again, this does not affect it.

Reply:
Ejection was a concept in Create React App since it hid a lot of the Webpack stuff from you and they gave you an escape hatch to go on your own. 

Ejection doesn't make sense when using Vite, because, Vite is the build tool, nothing is hidden.